### PR TITLE
BusinessDashboard Theme | Override Divider dark skin

### PR DESCRIPTION
**Pull request by liorf-wix**
Dashboard Theme - Add Dark Skin Divider to dashboard theme
Jira: https://jira.wixpress.com/browse/OSB-440

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [X] 👨‍🎨 UX change is approved by the Design System UX  - Ayelet Livne
- 📚 Change is documented
  - [X] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [X] Visual test

### CODE REVIEW

File: packages/wix-style-react/src/Themes/businessDashboard/components/Divider.st.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  -st-named: D50;
+}
+
+.root {
+  -st-extends: Divider;
+}
+
+.root:skin(dark) {
+  background-color: value(D50);
+}
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Divider.st.css:17]: add line break

File: packages/wix-style-react/src/Themes/businessDashboard/test/businessDashboard.visual.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import businessDashboardTheme from '../BusinessDashboard';
 import { runTests as ButtonTests } from '../../../Button/test/Button.visual';
 import { runTests as CircularProgressBarTests } from '../../../CircularProgressBar/test/CircularProgressBar.visual';
+import { runTests as DividerTests } from '../../../Divider/test/Divider.visual';
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/test/businessDashboard.visual.js:5]: @liorf-wix you need to implement the runTests in the Divider.visual

**Review by adicicurel — CHANGES_REQUESTED** [review]
@liorf-wix you need to implement the runTests in the Divider.visual

**Review by adicicurel — APPROVED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6946/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6946

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6946/

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6946

**Pull request merged**